functions/callPythonFunction/pythonFunctions/create_certificates.py

```
@@ -45,8 +45,28 @@ def __init__(self, arguments, enrollments=None, edu_hub_client=None):
+    def _is_degree_certificate(self):
+        """
+        Checks if this is a degree certificate by verifying if the program ID is 13.
+        
+        Returns:
+            bool: True if this is a degree certificate (Program ID 13), False otherwise
+        """
+        try:
+            if self.enrollments and len(self.enrollments) > 0:
+                program_id = self.enrollments[0].get('Course', {}).get('Program', {}).get('id')
+                return program_id == 2
+            return False
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)